modules/lss-shareAlliancePost/ShareAlliancePost.js

```diff
@@ -0,0 +1,260 @@
+                                    $('#allianceShareText').val(messages[extraKey - 1]);
+                                }
+                            }
+
+                            processAllianceShare();
+
+                        }
+                    }
+                });
+
+                $(document).keyup((e) => {
+                    keys.splice(keys.indexOf(e.which));
+                });
+            }
+        } catch (e) {
+            console.warn('Error initializing keys', e);
+        }
+    };
+
+    const processAllianceShare = () => {
+
+        $('#allianceShareOptions').hide();
+        $('#openAllianceShareOptions').show();
+
+        const sendToAlliance = $('#postToChat').is(':checked') ? 1 : 0;
+        const missionShareLink = $('#mission_alliance_share_btn').attr('href');
+        const missionId = missionShareLink.replace('/missions/', '').replace('/alliance', '');
+        const csrfToken = $('meta[name="csrf-token"]').attr('content');
+        const message = $('#allianceShareText').val();
+
+        $('.alert_notify_alliance').html(I18n.t('lssm.sharealliancepost.share'));
+        $.get('/missions/' + missionId + '/alliance', () => {
+            $('.alert_notify_alliance').html(I18n.t('lssm.sharealliancepost.chat'));
+            $.post("/mission_replies", { authenticity_token: csrfToken, mission_reply: { alliance_chat: sendToAlliance, content: message, mission_id: missionId } }, (data, status, xhr) => {
+                $('.alert_notify_alliance').html(I18n.t('lssm.sharealliancepost.alert'));
+                if (getSetting('jumpNext')) {
+                    $('.alert_next').first().click();
+                } else {
+                    $('#mission_alarm_btn').click();
+                }
+            });
+        });
+
+    };
+
+    const transformMessages = () => {
+        try {
+            // Prepare values for %ADDRESS% and %PATIENTS_LEFT%
+            // Possible inputs 'xy street, 1234 city', '1234 city', '123 city | 2' (where 2 is number of patients)
+            let addressAndPatrientRow = $('.mission_header_info >> small').first().text().trim().split(',');
+            addressAndPatrientRow = addressAndPatrientRow[addressAndPatrientRow.length - 1].split('|');
+            const address = addressAndPatrientRow[0];
+            const patientsLeft = addressAndPatrientRow.length === 2 ? addressAndPatrientRow[1] : 0;
+
+            // Prepare values for %MY_CUSTOM_TIME%
+            const offsetInHours = getSetting('timeOffset');
+            let customTime = new Date().getHours() + offsetInHours;
+            customTime = customTime > 24 ? customTime - 24 : customTime;
+
+            // Prepare required Vehicles
+            const alertText = $('.alert-danger');
+            const requiredVehiclesIdentifier = 'Zusätzlich benötigte Fahrzeuge:';
+            let requiredVehicles = 'Keine weiteren Fahrzeuge benötigt.';
```

Comment:
I18n auch hier

Reply:
you can edit it by yourself. Just by clicking on the edit icon by the code in the pull